src/it/scala/akka/kubernetes/sample/AkkaKubernetesSpec.scala

```diff
@@ -0,0 +1,73 @@
+package akka.kubernetes.sample
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{HttpRequest, StatusCodes}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.management.cluster.{ClusterHttpManagementJsonProtocol, ClusterMembers}
+import akka.stream.ActorMaterializer
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
+import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpec}
+
+class AkkaKubernetesSpec extends WordSpec with BeforeAndAfterAll with ScalaFutures with Matchers with ClusterHttpManagementJsonProtocol with Eventually {
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+
+  implicit override val patienceConfig = PatienceConfig(timeout = Span(30, Seconds), interval = Span(2, Millis))
+
+  val target = System.getProperty("akka.k8s.target", "http://localhost:8080")
+  val clusterSize = System.getProperty("akka.k8s.cluster-size", "1").toInt
+  val deployedVersion = System.getProperty("akka.k8s.deployment-version", "LOCAL")
```

Comment:
you could grab these from `system.settings.config` since system properties are also loaded in config

Reply:
good idea